python/setup.py

```diff
@@ -46,7 +46,7 @@ def read(name):
     extras_require={
         "full": ["numpy", "zarr"],
         "jupyter": ["numpy", "ipykernel"],
-        "socketio": ["numpy", "python-socketio[client]"],
+        "socketio": ["numpy", "python-socketio[client]==5.0.4", "python-engineio==4.0.0"],
```

Comment:
Why are we not using the latest version of python-socketio?

Reply:
Somehow we have some issues connecting to the server here: https://github.com/imjoy-team/imjoy-engine/pull/106

Reply:
Should we let imjoy-engine decide which version to use then? It is installing this package, right? We should just add socketio as a general requirement.

Reply:
```suggestion
        "socketio": ["numpy", "python-socketio[client]"],
```

Reply:
The issue is the rpc library may not run on the same machine as the imjoy-engine. To make sure they work with each other, they need to have the same library versions.

Reply:
What will run the rpc library in the other machine then?

Reply:
If the rpc package version is not the same in the two different machines we don't know that the two machines have the same pinning here anyway.

Reply:
At least we know the server will have the same version now. The issue is, if I just do without, I cannot get the connection work. I would pin it for now and explore later to see what the exact issue.

Reply:
But what is using this package in the server?

Reply:
the imjoy-rpc will be imported by imjoy.runner on an independent machine (other than the machine that runs imjoy.server).

Reply:
Would this be better?
```
"python-socketio[client]>=5.0.4",
"python-engineio>=4.0.0",
```

Reply:
So then we should pin or constrain the version in imjoy.runner and imjoy.server.

Reply:
Yes, a minimum version is ok here.

Reply:
In that case yes, but that won't cover the case when one just use imjoy_rpc directly like this:
```
from imjoy_rpc import connect_to_server, api

connect_to_server({"name": "my plugin", "server_url": "https://api.imjoy.io"})

class ImJoyPlugin:
    async def setup(self):
        pass

    async def run(self, ctx):
        await api.log("hello world.")

api.export(ImJoyPlugin(), config={"name": "test-plugin", "workspace": "123"})

```